src/Sylius/Component/Core/Updater/UnpaidOrdersStateUpdater.php

```diff
@@ -47,6 +50,18 @@ public function __construct(
             );
         }
 
+        if ($this->stateMachineFactory instanceof FactoryInterface) {
+            trigger_deprecation(
+                'sylius/core',
+                '1.13',
+                sprintf(
+                    'Passing an instance of "%s" as the first argument is deprecated. It will accept only instances of "%s" in Sylius 2.0.',
```

Comment:
```suggestion
                    'Passing an instance of "%s" as the second argument is deprecated. It will accept only instances of "%s" in Sylius 2.0.',
```